src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/gather/gather_kernel_ref.cpp

```diff
@@ -102,15 +147,14 @@ static inline std::vector<std::string> GetOrder(size_t size) {
     } else if (size == 6) {
         idx_order = {"b", "f", "w", "z", "y", "x"};
     }
-
     return idx_order;
 }
 
 static std::string GetDictionaryIndexOrder(const gather_params& params, size_t axis) {
-    std::vector<std::string> idx_order = GetOrder(params.outputs[0].GetDims().size());
+    const std::string& input_axis_index_macro = "INPUT_AXIS_INDEX";
+    const std::string& zero_val = "0";
```

Comment:
Why did you flip the sequence of these three lines?

Reply:
It didn't have any special meaning, just changed while modifying some codes. I will revert that changes soon.

Reply:
Done.